### PR TITLE
Fix a typo in the osu!catch RC article

### PR DESCRIPTION
**Pull request by osuZiRoX**
Fixes a mistake in a Platter guideline

---

### CODE REVIEW

**Comment by Okorin** [comment]
corresponding thread now at https://osu.ppy.sh/community/forums/topics/799913

slated merge date in 5 days

**Comment by osuZiRoX** [comment]
Yeah, it's a change that was brought in this thread: https://osu.ppy.sh/community/forums/topics/725256

You can check from Ascendance's latest post (https://osu.ppy.sh/community/forums/topics/725256?start=6610239) that it was supposed to say "hyperdashes in the guideline. You can see in the following posts that I messed up when changing the part that says x1.2 to x1.1 (how even?).

File: wiki/Ranking_Criteria/osu!catch/en.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ Difficulty-specific rules and guidelines do only apply to the difficulty level t
 #### Guidelines
 
 - **All distances should be clear on whether they require the player to walk or dash.** This is to ensure that players can easily recognize patterns that require dashing.
-- **Higher snapped dashes should not be followed by antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.1x the trigger distance and the movement after the dash must be walkable.
+- **Higher snapped hyperdashes should not be followed by antiflow patterns.** If used, the spacing must not exceed a distance snap of 1.1x the trigger distance and the movement after the hyperdashdash must be walkable.
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/osu!catch/en.md]: hyperdashdash?

Comment (osuZiRoX) [reply wiki/Ranking_Criteria/osu!catch/en.md]: I'm a retard.

**Pull request merged**